lean4.yaml

```diff
@@ -29,7 +29,18 @@ pipeline:
     with:
       repository: https://github.com/leanprover/lean4
       tag: v${{package.version}}
-      expected-commit: 11ccbced796476be020459a83c599b301a765d3e
+      expected-commit: 6caaee842e9495688c1567e78c0e68dbb96942aa
+
+  # The package ftbfs after the update of libuv to 1.51
+  - uses: patch
+    with:
+      patches: libuv-1.51.patch
+
+  # we really shouldn't be modifying stage0 files, which are used for
+  # bootstrapping, but I was unable to find a way to regenerate them
```

Comment:
Agreed with the comment.  Do we know if upstream is aware of these issues?

Reply:
Upstream uses clang during their release process, https://github.com/leanprover/lean4/blob/master/doc/make/index.md, which @dannf and I also tried switching too but we ended up with a different class of errors.